docs/Process_CI.md

```diff
@@ -73,7 +73,7 @@ to trigger the action.
 
 Generates all desktops for arm64 and x86 arhitecture to verify if they build correctly. Build is triggered every day, manually (by [any member of Armbian project](https://github.com/orgs/armbian/people)) or in pull requests if label "Desktop" is set. Aim of this test case is to find out if there are troubles in packages relations.
 
-- releases: bullseye, sid, jammy, focal,
+- releases: buster, bookworm, jammy,
```

Comment:
I assume this should be bullseye, bookworm and jammy.

Reply:
yes, thanks, fixed and squashed